shortbread-website/content/make-vectortiles/bbox.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Using BBOX"
+description = "Generating vector tiles with BBOX"
+weight = 4
++++
+
+Shortbread Tiles can be created with [BBOX](hhttps://www.bbox.earth/).
+
+## BBOX overview
+
+* Generate and serve vector tiles
+* OGC API services
+* Raster and vector tiles
+* Formats: PostGIS, MBTiles, PMTiles, S3
+* Non-Mercator tile grids
+* Successor of t-rex tile server
```

Comment:
I don't think we should mention OCG API, Raster, or non-mercator tile grids, but instead keep it focused to what is directly connected to shortbread

Reply:
I left the "Non-Mercator" sentence, because this is relevant for people importing OSM in a local projection.

Reply:
I agree with pnorman here. Creating non-mercator styles will probably need some changes to the shortbread styles, so this is quite a specialized thing. We should not make it look like this is an easy "just switch on non-mercator" thing.

Reply:
I know about a German city GIS departement using Shortbread in a local SRS with adapted styles using `osm2psql`. But no problem to remove that sentence to avoid false expectation.